source/embedded-shell.txt

```diff
@@ -20,8 +20,10 @@ to test queries and operations in your database.
 Open the Embedded MongoDB Shell
 -------------------------------
 
-To open the embedded :binary:`mongosh`, click :guilabel:`_MONGOSH`
-at the bottom of the |compass| window.
+To open the embedded :binary:`mongosh`, click :guilabel:`>_` on the
+right of your deployment name in the :guilabel:`Connections Sidebar` or
+click :guilabel:`>_Open MongoDB Shell` in the top right of the
+:guilabel:`Databases` tab. 
```

Comment:
[Optional suggestion]

To make this a bit more scannable, we could consider making these bullets. As in:

```
To open the embedded :binary:`mongosh`, you can either:

- Click :guilabel:`>_` on the right of your deployment name in the :guilabel:`Connections Sidebar`.
- Click :guilabel:`>_Open MongoDB Shell` in the top right of the :guilabel:`Databases` tab.

Reply:
Switched these lines to bullets! Thanks!